msteams-platform/concepts/deploy-and-publish/overview.md

```diff
@@ -1,56 +1,72 @@
 
-1. On the **Connect to channels** page, under **Add a featured channel**, select **Teams**.
-1. Navigate to the **Configure MSTeams** page (*see* below).
-1. Under **Messaging** select the **Microsoft Teams for Government Customers** radio button.
-1. In the lower left corner of the page, select **Save**.  
-
->[!IMPORTANT]
-> You cannot use the Teams commercial configuration to upload/sideload your custom app to a GCC environment,  You must select the **Microsoft Teams for Government Customers** radio button for a GCC compliant configuration.
+**To upload an app to your organization's environment**
 
+1. On the **Connect to channels** page, under **Add a featured channel**, select **Teams**.
+2. Navigate to the **Configure MSTeams** page.
+3. Under **Messaging**, select *Microsoft Teams for Government*.
 ![Teams messaging configuration page](../../assets/images/gcc-configure.png)
+   > [!IMPORTANT]
+   > You cannot use the *Microsoft Teams Commercial* configuration to upload or sideload your custom app to a GCC environment.
+4. In the lower left corner of the page, select **Save**.
 
 > [!NOTE]
->
-> * The uploading instructions for GCC environments, presented above, applies to Teams custom apps. </br>
-> * Compliant Microsoft apps are enabled in the GCC environment, by default, in Teams.
-> * Third-party apps are disabled at the tenant level and should be managed via your organization's [app permission policies](/microsoftteams/teams-app-permission-policies). Make certain that you review all third-party apps to ensure that they align with your organization's policies and procedures.
+> * The uploading instructions for the GCC environment also apply to Teams custom apps. </br>
+> * Only Microsoft compliant apps are permitted in the GCC environment and in Teams.
+> * Third-party apps are disabled at the tenant level and are managed through your organization's [app permission policies](/microsoftteams/teams-app-permission-policies). You must review all third-party apps to ensure they align with your organization's policies and procedures.
 
 > [!TIP]
->
-> Microsoft 365 developer partners provide security, data handling, and compliance details for their third-party Teams apps through the [Microsoft 365 App Certification Program](/microsoft-365-app-certification/overview). *See also* [Microsoft Teams App Certification](/microsoftteams/platform/concepts/deploy-and-publish/appsource/post-publish/application-certification).
-</br></br>
+> Microsoft 365 developer partners provide security, data handling, and compliance details for their third-party Teams apps through the [Microsoft 365 App Certification Program](/microsoft-365-app-certification/overview). For more information on Teams app certification, see [Microsoft Teams app certification](/microsoftteams/platform/concepts/deploy-and-publish/appsource/post-publish/application-certification).
```

Comment:
This seems like a related link. As a standalone tip, it does not seem connected to anything in the doc.